core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -15,7 +15,67 @@ import scala.concurrent.duration._
 
 /**
   * A stream producing output of type `O` and which may evaluate `F`
-  * effects. If `F` is [[Pure]], the stream evaluates no effects.
+  * effects.
+  * Or: a lazy sequence of computations that may evaluate `F` effects `F` and yield values of type `O`.
```

Comment:
Can we combine the first sentence with this one (the one that starts with Or) -- they aren't different enough to both exist IMO.

Reply:
:+1: I would advise against "sequence of computations" btw, it's subtle, but it can be misleading. There are some key differences between `Stream[F, A]` and `List[F[A]]`, for example the fact that you can always recover and continue from error given `List[F[A]]`, but you have to terminate the Stream with `Stream[F, A]`.

This is because subsequent computations in Stream might depend on previous ones monadically (from a data structure perspective, a Stream is closer to a tree of computations, given that it's a Free monad).

However, none of this should make into the scaladoc imho (you still want it to be concise and to the point), so I'd stick with "a computation emitting output of type `O` and evaluating `F` effects"

Reply:
Perhaps "lazy monadic chain of computations that evaluate effects in `F` and may yield `O` values"?
Monadic, of course, on the `FreeC`, not on the `F` itself.